resources/public/api/conf/common/testing.md

```diff
@@ -1,20 +1,6 @@
-You can use the sandbox environment to <a href="/api-documentation/docs/testing">test this API</a>. You can use the <a href="/api-documentation/docs/api/service/api-platform-test-user/1.0">Create Test User API</a> or it's frontend service to create test users.
+You can use the Sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing). You can test different scenarios in the Sandbox by passing the
```

Comment:
Environment specific url can be used `[test this API](/api-documentation/docs/testing)`